app.js

```diff
@@ -0,0 +1,75 @@
+import isYes from '../isYes.js'; 
```

Comment:
you will need to move your app.js and your isYes.js files into your src folder. 
Also, you should rename your three isYes functions to represent what the function does, and they are not returning a true or false result to wether or not the question was answered with a yes or no